test/end2end/aws_environment.py

```diff
@@ -0,0 +1,15 @@
+"""AWS environment."""
```

Comment:
Can you please delete the file or write a comment here that it's just a placeholder?